website/docs/language/functions/filesha256.mdx

```diff
@@ -5,11 +5,46 @@ description: |-
   a given file and encodes it as hex.
 ---
 
-# `filesha256` Function
+# `filesha256` function reference
+This topic provides reference information about the `filesha256` function, which calculates the SHA-256 hash of a file's contents. 
+
+## Introduction
 
 `filesha256` is a variant of [`sha256`](/terraform/language/functions/sha256)
 that hashes the contents of a given file rather than a literal string.
 
 This is similar to `sha256(file(filename))`, but
 because [`file`](/terraform/language/functions/file) accepts only UTF-8 text it cannot be used to
 create hashes for binary files.
```

Comment:
```suggestion
Use the `filesha1` function instead of wrapping the `file` function in a `sha1` function, for example `sha1(file(filename))`, because [`file`](/terraform/language/functions/file) accepts only UTF-8 text. As a result, you cannot use `sha1(file(filename))`  to create hashes for binary files.
```